test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -298,12 +287,11 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
             helper.Process(context, output);
 
             // Assert
-            Assert.Null(output.TagName);
```

Comment:
No. This will be `null` in the case when `Href` is not specified but `HrefInclude` is specified.